File: Fibonacci/questao1.py
def pertence_fibonacci(numero):
    a, b = 0, 1
    while b < numero:
        a, b = b, a + b
    return b == numero or numero == 0

numero = 21  # Digite o número para conferir
print(f"O número {numero} {'pertence' if pertence_fibonacci(numero) else 'não pertence'} à sequência de Fibonacci.")

File: Fibonacci/questao1.js

function pertenceFibonacci(numero) {
    let a = 0, b = 1, temp;
    while (b < numero) {
        temp = a;
        a = b;
        b = temp + b;
    }
    return b === numero || numero === 0;
}

let numero = 21; // Digite o número para conferir
console.log(`O número ${numero} ${pertenceFibonacci(numero) ? 'pertence' : 'não pertence'} à sequência de Fibonacci.`);
